docs/DifferentiableProgrammingImplementation.md

```diff
@@ -0,0 +1,1003 @@
+# Swift Differentiable Programming Implementation Overview
+
+- Authors: [Dan Zheng](https://github.com/dan-zheng), [Bart Chrzaszcz](https://github.com/bartchr808), [Philip Turner](https://github.com/philipturner)
```

Comment:
Have you made extensive changes to the document that from the original document?   I'm asking as you are marking yourself as an author, and I'd like to understand what the difference is from the original document being converted.

Reply:
Look at my PR to swift-apis (hyperlinked in the first comment to this PR). I outlined 5 changes I made to the original document. @compnerd please look over the other 4 and validate them.

Reply:
We really appreciate your contribution to this document, but the authorship in this document was meant to reflect substantial _technical_ contribution from years of design and development in addition to just writing.  As a result, I believe it's more appropriate to keep the original author list.  Be assured that your contribution will still be acknowledged, regardless of being listed as an author in the document.  Thanks!

Reply:
```suggestion
- Authors: [Dan Zheng](https://github.com/dan-zheng), [Bart Chrzaszcz](https://github.com/bartchr808)
```